github-workflows/build-latex-diff.yml

```diff
@@ -17,11 +17,12 @@ jobs:
       - name: Download LaTeX styles
         run: |
           curl -OL https://github.com/dpo/latex-styles/archive/refs/heads/master.zip
-          curl -OL https://www.siam.org/Portals/0/Macros/Standard/siamart_190516.zip
+          curl -OL https://epubs.siam.org/pb-assets/macros/standard/siamart220329.cls
```

Comment:
I prefer if we download the zip, so the method is the same for all styles. Users are also more likely to have the zip.